test/e2e/config/e2e_conf.yaml

```diff
@@ -129,6 +129,7 @@ variables:
   UPGRADED_BMO_IMAGE_TAG: "${UPGRADED_BMO_IMAGE_TAG:-main}"
 
   INIT_WITH_BINARY: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_FROM_RELEASE}/clusterctl-{OS}-{ARCH}"
+  PROVIDER_ID_FORMAT: "metal3://{{ ds.meta_data.providerid }}"
```

Comment:
Can you explain a bit more about this addition?

Reply:
management upgrade test init the cluster with v1a4 which use the uuid format for the provider I am setting the v1b1 as default format here so after cleaning the management upgrade I just unset the env var and the other tests would use this default format